.travis.yml

```diff
@@ -118,6 +118,10 @@ before_install:
     fi
 install:
   - if [[ "${TEST_SAGE}" != "true" ]]; then
+      pip uninstall -y setuptools;
```

Comment:
Please add a comment above "install" explaining why this is done, otherwise people won't understand it.

Also, we need to make sure that after you call `pip uninstall -y setuptools` a second one isn't installed in the apt-get installs. Do something like `python -c "import setuptools"` to see if it errors.